File: backend/app/constants/cache.py
"""
Cache TTL constants (seconds).

Only values that appear as anonymous magic numbers in multiple places belong here.
Route-specific named TTLs (_LIST_CACHE_TTL, _SIM_CACHE_TTL, etc.) stay local.
"""

# Reference data — static game content, changes only on game patch
REF_STATIC_CACHE_TTL: int = 86400       # 24 hours

# Reference data — semi-static content (affixes, passives)
REF_SEMISTATIC_CACHE_TTL: int = 3600    # 1 hour

File: backend/src/constants/cache.ts
/**
 * Cache TTL constants (seconds).
 * Only values used in multiple places belong here.
 */

/** Reference data — static game content, changes only on game patch (24 hours) */
export const REF_STATIC_CACHE_TTL = 86400 as const;

/** Reference data — semi-static content, e.g. affixes, passives (1 hour) */
export const REF_SEMISTATIC_CACHE_TTL = 3600 as const;
